gslib/tests/util.py

```diff
@@ -665,7 +665,14 @@ def __init__(self, is_upload, halt_at_byte):
 
   # pylint: disable=invalid-name
   def call(self, total_bytes_transferred, total_size):
-    """Forcibly exits if the transfer has passed the halting point."""
+    """Forcibly exits if the transfer has passed the halting point.
+
+    Note that this function is only called when the linked conditions are
+    met: https://github.com/GoogleCloudPlatform/gsutil/blob/master/gslib/progress_callback.py#L128.
```

Comment:
Seems weird to HTTP link something in the same codebase. Maybe just give the file path?
I wouldn't provide the line number b/c it might change.